torch/csrc/jit/ir.cpp

```diff
@@ -250,11 +256,20 @@ std::ostream& printNode(std::ostream & out, const Node * n, std::vector<const No
     out << ", ";
     out << "scope: " << scopeName << "\n";
   }
+  for(size_t i = 0; i < n->blocks().size(); ++i) {
+    auto b = n->blocks()[i];
+    indent(out, level + 1) << "block" << i << "(" << const_value_list_with_types(b->inputs(), false) << ") {\n";
```

Comment:
Printing them before the node would be quite weird

Reply:
Im not too worried about that because in a functional language you write:

```
a = if(b) {
    b +  1
} else {
   3
};
```

and `a` is not in scope in the body.